python-threatexchange/threatexchange/signal_type/url.py

```diff
@@ -40,3 +40,13 @@ def matches_str(
     @staticmethod
     def get_examples() -> t.List[str]:
         return ["https://developers.facebook.com/docs/threat-exchange/reference/apis/"]
+
+    @classmethod
+    def normalize_fb_threatexchange_indicator(
```

Comment:
Did this sneak in from a different soon to be PR?

Reply:
Stacked, should have noted